package/icu/icu.mk

```diff
@@ -4,28 +4,21 @@
 #
 ################################################################################
 
-ifeq ($(BR2_PACKAGE_RDK_VERSIONING),y)
-ICU_VERSION = 57.1
-else ifeq ($(BR2_PACKAGE_NETFLIX5),y)
 ICU_VERSION = 65.1
-else ifeq ($(BR2_PACKAGE_NETFLIX52),y)
-ICU_VERSION = 65.1
-else
-ICU_VERSION = 57.1
-endif
 
 ICU_SOURCE = icu4c-$(subst .,_,$(ICU_VERSION))-src.tgz
-ifeq ($(ICU_VERSION), 65.1)
 ICU_SITE = \
 	https://github.com/unicode-org/icu/releases/download/release-$(subst .,-,$(ICU_VERSION))
 ICU_LICENSE = ICU License
 ICU_LICENSE_FILES = LICENSE
+
+ICU_FILTERS_DATA_PATH = $(call qstrip,$(BR2_PACKAGE_ICU_FILTERS_DATA_PATH))
+# Force to build icudate, as of ICU 64 the website to generate icudate is no longer applicable
+# If someone needs to build it outside still mk file is able to support it
 ICU_BUILD_ICUDATA = y
```

Comment:
icudata.mk and Config.in may also be modified as it is always building icudata and not using pre-generated icudata from Metrological repository.